roles/get_image_hash/tasks/get_image_hash.yml

```
@@ -2,7 +2,7 @@
   when: (item.value.hash | default('')) == ''
   block:
   - name: "Get {{ item.key }} image hash"
```

Since this is being change, could you rename it to:
```suggestion
  - name: "Get image hash - {{ item.key }}"
```